features/step_definitions/assertion_steps.rb

```diff
@@ -1,6 +1,10 @@
 Then(/^I should see an image named "([^"]*)"$/) do |image_name|
+  image = Image.find_by(name: image_name)
   within '#images' do
```

Comment:
You probably don't need both. Since `.image-12` or whatever is specific to each image, you can just search for that div directly.